source/numerics/vector_tools_project.inst.in

```diff
@@ -67,3 +67,27 @@ for (VEC : SERIAL_VECTORS ; deal_II_dimension : DIMENSIONS; deal_II_space_dimens
     \}
 #endif
 }
+
+for (VEC: REAL_NONBLOCK_VECTORS; deal_II_dimension : DIMENSIONS; deg : DEGREES)
```

Comment:
@tamiko: any way to specify `DEGREES` here in `.inst.in` as opposed to `template-arguments.in`?